src/summary/documentation/vitals-card.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import { match } from "react-router";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+
+export default function VitalsCard(props: VitalsCardProps) {
```

Comment:
Are you planning on following up this PR with another one that actually fetches the vitals from the API and displays them correctly? If so, that would be a good thing to indicate in your pull request's description or in a comment, so that reviewers understand how/where your code is incomplete.